# EUKARYOME

Accurate taxonomic identification of organisms is one of the cornerstones of biology and its many subdisciplines, 
including ecology, biogeography, phylogenetics, and conservation biology. 
Molecular identification of organisms based on nuclear markers has revolutionised our understanding about 
the taxonomy, phylogeny, and ecology of various microscopic and macroscopic organisms. 
**EUKARYOME is a community-curated research database** for nuclear ribosomal 18S rRNA (SSU), 
internal transcribed spacer (ITS), and 28S rRNA (LSU) markers **for all eukaryotes**.  

The key benefits of this database are:  

- It is not restricted to some taxonomic groups and it includes all rRNA markers;  
- EUKARYOME compiles well-annotated, non-redundant, high-quality reads for the SSU, ITS, and LSU subsets – separately and combined – for high-accuracy taxonomic reference and chimera recognition;  
- EUKARYOME also offers a number of reference long read sequences that are derived from (meta)genomic, long-read (meta)barcoding and concatenated short Sanger reads – a unique feature that can be used for taxonomic identification and chimera control of third-generation, long-read, high-throughput sequencing data.  

EUKARYOME relies only on the Linnaean ranks including species, genus, family, order, class, phylum, and kingdom. 
The EUKARYOME reference database covers all terrestrial and aquatic eukaryotic groups based on nuclear rRNA 
SSU (150,442 reads), ITS (1,111,297), and LSU (100,197) markers taken separately and combined (28,684 SSU-ITS-LSU reads). 
A total of 1,244,454 entries are contained in the current version of the database.  
